BrowserSelect/Form1.cs

```diff
@@ -201,7 +201,9 @@ private void browser_click(object sender, EventArgs e)
 
         public static void open_url(Browser b, bool incognito = false)
         {
-            var args = new List<String>();
+            var args = new List<string>();
+            if (b.exec.ToLower().EndsWith("chrome.exe") && !String.IsNullOrEmpty(b.additionalArgs))
```

Comment:
why check if it's chrome? always appending additional args if not null or empty doesn't seem to cause any side effects and could be useful in the future.

Reply:
didn't want to mess up things. But you're completely right. As the string is always initialized at least with an empty string, this should make no difference.